routes/car.js

```diff
@@ -1,8 +1,81 @@
 const express = require('express');
 const router = express.Router();
+const Car = require('../models/car');
+const User = require('../models/user');
+const jwtDecode = require('jwt-decode');
 
 router.get('/', (req, res) => {
-  res.send('car Route is working')
+    res.send('car Route is working')
 });
 
+
+router.post('/add', async (req, res) => {
+    const payload = jwtDecode(req.header('authorization'));
```

Comment:
it's likely we just pass them all  by request body.
You can just do `const { userId, payload } = req.body`
then use payload.license, etc.
You don't need jwtDecode

Reply:
Although not critical in terms of the scope of the project, passing in the `userId` via  ``

Reply:
The `userId` that we are passing is not the username. It's the mongoDB document id in ObjectID form.
There's no security risk to it.